dataclients/kubernetes/kubernetestest/api.go

```diff
@@ -329,6 +331,7 @@ subsets:
 `
 
 func getJSON(u string, o interface{}) error {
+	/* #nosec */
```

Comment:
I wonder why we need this for the GET request. Maybe we do something wrong.

Reply:
I missed that this is inside the kubernetestest package and @muaazsaleem clarified that this gosec check was newly hit, because file move from api_test.go to api.go, so it's fine to ignore here.